File: codigos-java/SistemaDeNotas.java
import java.util.Scanner;

public class SistemaDeNotas {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        double nota, soma = 0.0, media = 0.0;
        String login;
        int senha, n;

      
        System.out.println("Digite seu login, somente letras: ");
        login = scanner.nextLine();

       
        System.out.println("Digite sua senha, somente números: ");
        senha = scanner.nextInt();
        scanner.nextLine(); 

      
        if (login.equals("Rafael") && senha == 123456) {
            System.out.println("Acesso foi feito com sucesso!");

          
            System.out.println("Informe quantas notas você deseja informar: ");
            n = scanner.nextInt();

            
            for (int i = 0; i < n; i++) {
                System.out.println("Informe a " + (i + 1) + "° nota: ");
                nota = scanner.nextDouble();

                while (nota < 0.0 || nota > 10.0) {
                    System.out.println("Nota inválida! Informe uma nota entre 0 e 10.");
                    nota = scanner.nextDouble();
                }

                soma += nota;
            }

           
            media = Math.round((soma / n) * 100.0) / 100.0;
            System.out.println("A média das notas é: " + media);
        } else {
            System.out.println("Acesso negado");
        }

        scanner.close();
    }
}

File: Códigos C/SistemaDeNotas.c
#include <stdio.h>
#include <string.h>

int main() {
    char login[50];
    int senha, n, i;
    double nota, soma = 0.0, media = 0.0;

    printf("Digite seu login, somente letras: ");
    scanf("%s", login);

    printf("Digite sua senha, somente números: ");
    scanf("%d", &senha);

    if (strcmp(login, "Rafael") == 0 && senha == 123456) {
        printf("Acesso foi feito com sucesso!\n");

        printf("Informe quantas notas você deseja informar: ");
        scanf("%d", &n);

        for (i = 0; i < n; i++) {
            printf("Informe a %d° nota: ", i + 1);
            scanf("%lf", &nota);

            while (nota < 0.0 || nota > 10.0) {
                printf("Nota inválida! Informe uma nota entre 0 e 10.\n");
                scanf("%lf", &nota);
            }

            soma += nota;
        }

        media = soma / n;
        printf("A média das notas é: %.2f\n", media);
    } else {
        printf("Acesso negado\n");
    }

    return 0;
}
